tensorflow/python/ops/math_ops.py

```diff
@@ -1577,6 +1577,25 @@ def sigmoid(x, name=None):
     return gen_math_ops._sigmoid(x, name=name)
 
 
+def sigmoid_grad(x, name=None):
+  """Computes the gradient of the sigmoid of `x` element-wise.
+
+  Specifically, `y = sigmoid(x) * (1 - sigmoid(x))`.
+
+  Args:
+    x: A Tensor with type `float32`, `float64`, `int32`, `complex64`, `int64`,
+      or `qint32`.
+    name: A name for the operation (optional).
+
+  Returns:
+    A Tensor with the same type as `x` if `x.dtype != qint32`
+      otherwise the return type is `quint8`.
+  """
+  with ops.name_scope(name, "SigmoidGrad", [x]) as name:
+    x = ops.convert_to_tensor(x, name="x")
+    return gen_math_ops._sigmoid_grad(x, name=name)
+
```

Comment:
btw, there's also `tf.square(x)`, I'm guessing that one avoids allocating `x` twice internally like `x*x`